modules/services/cliphist.nix

```diff
@@ -17,6 +17,22 @@ in {
       '';
     };
 
+    maxDedupeSearch = lib.mkOption {
+      type = lib.types.int;
+      default = 100;
+      description = ''
+        Maximum number of last items to look through when finding duplicates.
+      '';
+    };
+
+    maxItems = lib.mkOption {
+      type = lib.types.int;
+      default = 750;
+      description = ''
+        Maximum number of items to store.
+      '';
+    };
```

Comment:
I think instead of adding options for every comment line flag we should do something like [`programs.atuin.flags`](https://github.com/nix-community/home-manager/blob/master/modules/programs/atuin.nix#L54)